espresso-server/app/src/androidTest/java/io/appium/espressoserver/EspressoServerRunnerTest.kt

```diff
@@ -80,6 +87,16 @@ class EspressoServerRunnerTest {
         assertEquals(true, true) // Keep Codacy happy
     }
 
+    private fun disableCaptureScreenshotOnFailureByEspressoLib() {
+        val constructors = DefaultFailureHandler::class.constructors
+        val disableScreenShotOnFailure = constructors.any { it.parameters.last().name.toString() == "captureScreenshotOnFailure" }
+        if (disableScreenShotOnFailure) {
+            AndroidLogger.info("`DefaultFailureHandler` has `captureScreenshotOnFailure` parameter which is set to `true` by default" +
+            "Setting it to `false` to fix slow espresso test run" + "\nFixes https://github.com/android/android-test/issues/1801")
```

Comment:
do we need to catenate strings here with `+`?

Reply:
Fixed